docs/UserGuide.md

```diff
@@ -200,6 +200,38 @@ If `/edit` is supplied instead of a note, (e.g. `note 1 /edit`), the contents of
 > Executing the command `note 1 /edit` will replace the contents of the command box with `note 1 Wants to gain muscle`.
 <hr>
 
+### Adding or overriding exercise(s) of clients : `fitadd`
+
+Format: `fitadd INDEX [n/EXERCISE_NAME] [s/SETS] [r/REPS] [b/BREAK_BETWEEN_SETS_IN_SECONDS] [/arms] [/legs] [/chest] [/back] [/shoulders] [/abs] [/all] `
+
+* Adds the specified exercise(s) to the client specified by `INDEX`. The index refers to the index number shown in the displayed client list. The index **must be a positive integer** 1, 2, 3, …​
+* If an exercise with same name already exists for the client, the old exercise will be overwritten with the newly supplied field values, or a default set of values if not supplied.
+* The `n/EXERCISE_NAME`, `s/SETS`, `r/REPS` and `[b/BREAK_BETWEEN_SETS_IN_SECONDS]` may be omitted in place of a default supported prefix such as `/arms` and `/legs`.
+* Using default supported prefixes such as `/arms` or `/legs` will add a pre-defined set of related exercises to the specified client.
+* You must either specify a specific exercise name or use a number of default supported prefixes, but not both together. 
+
+Examples:
+
+* `fitadd 1 n/burpees` - Adds or overwrites the `burpees` exercise of the 1st client with a default set of 1, repetition of 1 and 0 seconds break time between sets.
+* `fitadd 1 n/burpees s/3 r/5 b/30` - Adds or overwrites the `burpees` exercise of the 1st client with sets of 3, repetitions of 5 and 30 seconds break time between sets.
+* `fitadd 2 /arms` - Adds or overwrites a default set of exercises from the `arms` category to the 2nd client.
+* `fitadd 2 /arms /legs` - Adds or overwrites a default set of exercises from the `arms` and `legs` category to the 2nd client.
+<hr>
+
+### Deleting exercise(s) of clients : `fitdelete`
+
+Format: `fitdelete INDEX [n/EXERCISE_NAME] [/all]`
```

Comment:
Same as `fitadd` section

Reply:
Technically speaking it is.

You must either specify a specific exercise name or the `/all` prefix, but not both together.
Not sure how to represent that.